.azure-pipelines/upload-junit.ps1

```diff
@@ -0,0 +1,43 @@
+$resultFolder = "./tracer/build_data/results";
```

Comment:
We could pretty easily replace this whole file with a Nuke task I think. I think that would be preferable? It would simplify things too (e.g. we can auto-detect arch etc?)